datastore/cloud-client/snippets.py

```diff
@@ -150,26 +150,26 @@ def incomplete_key(client):
 
 def named_key(client):
     # [START datastore_named_key]
-    key = client.key("Task", "sample_task")
+    key = client.key("Task", "sampleTask")
```

Comment:
LGTM! Could we add a test somewhere for these?

Reply:
Thanks. I've added tests to assert that the `key.name == "sampleTask"`.